app/components/CrashNarrativeCard.tsx

```diff
@@ -0,0 +1,58 @@
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+import { FaFilePdf } from "react-icons/fa6";
+import AlignedLabel from "@/components/AlignedLabel";
+import { Crash } from "@/types/crashes";
+import { useToken } from "@/utils/auth";
+
+/**
+ * Card component that renders the crash narrative and a download CR3 button
+ */
+export default function CrashNarrativeCard({ crash }: { crash: Crash }) {
+  const token = useToken();
+
+  const isCr3Stored = crash.cr3_stored_fl;
+
+  // Downloads pdf from the CR3 API and opens it in a new tab
+  const onDownloadCR3 = async () => {
+    const requestUrl = `${process.env.NEXT_PUBLIC_CR3_API_DOMAIN}/cr3/download/${crash.record_locator}`;
+
+    try {
+      const response = await fetch(requestUrl, {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      });
+
+      if (!response.ok) {
+        const responseText = await response.text();
+        console.error(responseText);
+        window.alert(`Failed to download CR3: ${String(responseText)}`);
+      } else {
+        const responseJson = await response.json();
+        const win = window.open(responseJson.message, "_blank");
```

Comment:
you can test this error message by deleting your token from the fetch command. open to suggestions on better error messages :shrug:

Reply:
I think this is fine 👍